service/mantle/fedex/FedexServices.xml

```diff
@@ -0,0 +1,155 @@
+    <service verb="validate" noun="PostalAddress">
+        <implements service="mantle.shipment.CarrierServices.validate#ShippingPostalAddress"/>
+        <actions>
+            <!-- If a party/facility is expired(If replacesContactMechId is set) then address is not validated -->
+            <if condition="partyId">
+                <then>
+                    <entity-find entity-name="mantle.party.contact.PartyContactMechInfo" list="replacesPcmiList">
+                        <date-filter/><econdition field-name="partyId"/>
+                        <econdition field-name="replacesContactMechId" from="contactMechId"/>
+                        <select-field field-name="contactMechId"/><order-by field-name="contactMechId"/>
+                    </entity-find>
+                    <if condition="replacesPcmiList">
+                        <set field="contactMechId" from="replacesPcmiList[0].contactMechId"/><return/>
+                    </if>
+                </then>
+                <else-if condition="facilityId">
+                    <entity-find entity-name="mantle.facility.FacilityContactMechInfo" list="replacesFcmiList">
+                        <date-filter/><econdition field-name="facilityId"/>
+                        <econdition field-name="replacesContactMechId" from="contactMechId"/>
+                        <select-field field-name="contactMechId"/><order-by field-name="contactMechId"/>
+                    </entity-find>
+                    <if condition="replacesFcmiList">
+                        <set field="contactMechId" from="replacesFcmiList[0].contactMechId"/><return/>
+                    </if>
+                </else-if>
+            </if>
+
+            <!-- Converting Address Data to JSON Request Map -->
+            <service-call name="mantle.fedex.FedexServices.get#AddressMap" in-map="[postalContactMechId:contactMechId]"
+                out-map="addressResponse"/>
+            <if condition="!addressResponse.addressMap">
+                <return type="danger" message="Invalid address or contact details."/>
+            </if>
+            <set field="requestMap" from="[addressesToValidate:[[address:addressResponse.addressMap.address]]]" type="NewMap"/>
+
+            <!-- Setting Connection With Fedex -->
+            <service-call name="mantle.fedex.FedexServices.send#FedexRequest" out-map="response"
+                in-map="[shippingGatewayConfigId:shippingGatewayConfigId, method:'POST', path:'address/v1/addresses/resolve',
+                contentType:'application/json', requestMap:requestMap]"/>
+
+            <!-- Verifying Correctness of Address -->
+            <set field="responseMap" from="response.responseMap"/>
+            <set field="resolvedAddresses" from="responseMap?responseMap.output.resolvedAddresses[0]:null"/>
+            <set field="statusCodes" from="resolvedAddresses?resolvedAddresses.attributes:null"/>
```

Comment:
Why the name statusCodes? I think it is not status codes which are getting returned here.